### DISCUSSION TITLE
tweek to start orientation of Rg current buffer

### DISCUSSION THREAD

**Discussion by TimelordUK** [ibhagwan/fzf-lua#1189] [Q&A] [answered]
this is a great tool

So I have a quick question, this is my config thus far 

https://github.com/TimelordUK/configs/blob/main/.config/nvim/lua/plugins/fzf-lua.lua

for my keybinding fb I would like to start with equivalent of f5 i.e.  like below - but I have to press f5 each time, i did try changinging a few things so far to no avail.  I want equivalent of fb to open and F5 to be done autimatically for this particular command. I also added based on documents a zoxide integration ctrl-Z which works well for me.

can someone also tell me in Rg section can i quickly go to bottom or top - i would like to use this tool amongst many things to whiz through logs finding things of interest.

I did try read documentation, im sure i have missed things,

![image](https://github.com/ibhagwan/fzf-lua/assets/7300251/d767443d-a34e-404e-8664-2544cf1d39d6)

**Comment by ibhagwan** [comment, marked as answer]
> for my keybinding fb I would like to start with equivalent of f5 i.e. like below - but I have to press f5 each time

change layout from `flex` to  `vertical`.

> can someone also tell me in Rg section can i quickly go to bottom or top - i would like to use this tool amongst many things to whiz through logs finding things of interest.

Open `man fzf` and search for “available actions”, sounds like you want `first` and `last`, you can then map these under `keymap.fzf`:
```lua
require'fzf-lua'.setup({
  keymap = {
    fzf = {
      -- this table overrides the defaults so make sure to copy all other fzf keybinds you want here
      ['alt-g'] = 'first',
      ['shift-g'] = 'last',
    }
  }
})
```

Defaults are here:
https://github.com/ibhagwan/fzf-lua/blob/9c953dfa7650191d892800333d187f3439e01afd/lua/fzf-lua/defaults.lua#L82-L97

**Reply by TimelordUK** [reply]
thanks i thought i had tried this, indeed you are of course correct, its working now

im using tool all the time very useful indeed

**Reply by ibhagwan** [reply]
Ty for the update @TimelordUK
